src/lwc/utilInput/utilInput.js

```diff
@@ -169,38 +204,63 @@ export default class utilInput extends LightningElement {
         return undefined;
     }
 
-    get uiObjectApiName() {
-        return this.objectInfo && this.objectInfo.apiName ? this.objectInfo.apiName : this.objectApiName;
-    }
-
     get showRichTextLabel() {
-        return this.isLightningRichText && this.variant !== 'label-hidden';
+        return this.isLightningRichText && this.variant !== LABEL_HIDDEN;
     }
 
     @wire(getObjectInfo, { objectApiName: '$objectApiName' })
     wiredObjectInfo(response) {
         if (response.data) {
             this.objectInfo = response.data;
+            this.utilDescribe.setDescribe(response.data);
+            this.defaultRecordTypeId = this.utilDescribe.defaultRecordTypeId();
+            this.fieldDescribe = this.utilDescribe.getFieldDescribe(this.fieldApiName);
```

Comment:
It looks like `objectInfo` is no longer declared or used and might be able to be removed.  

`defaultRecordTypeId` and `fieldDescribe` might better be represented as function expressions since both are derived from the new utilDescribe object and could then be referenced as `this.defaultRecordTypeId()` and `this.fieldDescribe()`